packages/vx-geo/src/util/additionalProps.js

```diff
@@ -0,0 +1,8 @@
+import callOrValue from './callOrValue';
+
+export default function additionalProps(restProps, data) {
+  return Object.keys(restProps).reduce((ret, cur) => {
+    ret[cur] = callOrValue(restProps[cur], data);
+    return ret;
+  }, {});
+}
```

Comment:
this comment is just a reminder for myself to follow up on https://github.com/hshoff/vx/issues/69